misc/python/materialize/checks/all_checks/optimizer_notices.py

```diff
@@ -9,13 +9,14 @@
 from textwrap import dedent
 
 from materialize.checks.actions import Testdrive
-from materialize.checks.checks import Check
+from materialize.checks.checks import Check, exclude_from_parallel_execution
 from materialize.checks.executors import Executor
 from materialize.mz_version import MzVersion
 
 SCHEMA = "optimizer_notices"
 
 
+@exclude_from_parallel_execution(True)
```

Comment:
Why do these specific checks have to be excluded?

Reply:
These two checks change the `enable_session_rbac_checks` flag. As mentioned in the PR description, this PR

> is an alternative to https://github.com/MaterializeInc/materialize/pull/29752, which does not work.

I will add a comment to the exclude annotation if we want to merge this PR. Though, happy to try to get https://github.com/MaterializeInc/materialize/pull/29752 working if you have further ideas.